### DISCUSSION TITLE
[Maybe bug] Saving before there is one created contact

### DISCUSSION THREAD

**Discussion by inson1** [FossifyOrg/General-Discussion#677] [Q&A]
"I didn't have the option to save contacts to DEVICE until I created one contact through the system app"

https://github.com/SimpleMobileTools/Simple-Contacts/issues/1041#issuecomment-1789672742

May somebody try it?

**Comment by jhnc** [comment]
It's still the same with version from today.

**Comment by Aga-C** [comment]
It's reported here: https://github.com/FossifyOrg/Contacts/issues/186.
